portal/server/controllers/dashboard/facilities/facilities-filters-query.test.js

```diff
@@ -113,6 +113,69 @@ describe('controllers/dashboard/facilities - filters query', () => {
     expect(result).toEqual(expected);
   });
 
+  it(`should return hasBeenIssued filter when stage is ${DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.FACILITIES.ISSUED}`, () => {
+    const mockFilters = [
+      {
+        [CONSTANTS.FIELD_NAMES.FACILITY.STAGE]: [DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.FACILITIES.ISSUED],
+      },
+    ];
+
+    const result = dashboardFacilitiesFiltersQuery(mockFilters, mockUser);
+
+    const expected = {
+      AND: [
+        { 'deal.bank.id': mockUser.bank.id },
+        {
+          OR: [{ hasBeenIssued: true }],
+        },
+      ],
+    };
+
+    expect(result).toEqual(expected);
+  });
+
+  it(`should return hasBeenIssued filter when stage is ${DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.FACILITIES.UNISSUED}`, () => {
+    const mockFilters = [
+      {
+        [CONSTANTS.FIELD_NAMES.FACILITY.STAGE]: [[DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.FACILITIES.UNISSUED]],
```

Comment:
Are we able to destructure so we don't have `DASHBOARD_FILTERS.BESPOKE_FILTER_VALUES.FACILITIES.UNISSUED` in several places?